main.css

```
@@ -0,0 +1,69 @@
+*{

```

Для удобности чтения лучше использовать следующий формат
```suggestion
* {
```